### PR TITLE
Moved loadObjects to a registerObjects method

### PR DESCRIPTION
**Pull request by arthuro555**
To make clean dynamic object creation through JS possible.

### CODE REVIEW

**Comment by arthuro555** [comment]
Ready for review @4ian .

File: GDJS/Runtime/runtimescene.js [hunk 1]
```diff
@@ -156,6 +144,18 @@ gdjs.RuntimeScene.prototype.loadFromScene = function(sceneData) {
     this._timeManager.reset();
 };
 
+/**
+ * Registration of {@link gdjs.RuntimeObject}.
```
Comment (4ian) [inline GDJS/Runtime/runtimescene.js]: Start description with a verb describing the action done by the function.

File: GDJS/Runtime/runtimescene.js [hunk 2]
```diff
@@ -156,6 +144,18 @@ gdjs.RuntimeScene.prototype.loadFromScene = function(sceneData) {
     this._timeManager.reset();
 };
 
+/**
+ * Registration of {@link gdjs.RuntimeObject}. 
+ * This step is needed for any object before adding Instances of it to the scene.
+ * @param {ObjectData} objectData The objectData of the object to register.
```
Comment (4ian) [inline GDJS/Runtime/runtimescene.js]: @param {ObjectData} objectData The data for the object to register.

File: GDJS/Runtime/runtimescene.js [hunk 3]
```diff
@@ -156,6 +144,18 @@ gdjs.RuntimeScene.prototype.loadFromScene = function(sceneData) {
     this._timeManager.reset();
 };
 
+/**
+ * Registration of {@link gdjs.RuntimeObject}. 
+ * This step is needed for any object before adding Instances of it to the scene.
```
Comment (4ian) [inline GDJS/Runtime/runtimescene.js]: I would go for:
`Call this function before adding instances of an object to the scene.`

File: GDJS/Runtime/runtimescene.js (hunk 1 above)
Comment (4ian) [reply GDJS/Runtime/runtimescene.js]: `Register an object so that instances of it can be used in the scene`.

**Comment by 4ian** [comment]
Thanks, looks good :)

**Pull request merged**